flink-libraries/flink-table/src/main/java/org/apache/calcite/sql/fun/SqlGroupFunction.java

```diff
@@ -50,7 +50,8 @@
  * GROUP BY HOP(rowtime, INTERVAL '1' HOUR), productId
  * </pre></blockquote>
  */
-class SqlGroupFunction extends SqlFunction {
+// FLINK QUICK FIX
+public class SqlGroupFunction extends SqlFunction {
```

Comment:
Yes, I will open a issue and PR for Calcite today. I think visibility changes should not be a big problem for the Calcite folks.